guides/stax-delete.dockerfile

```
@@ -0,0 +1,73 @@
+FROM homebrew/brew
+
+RUN brew install TarasMazepa/stax/stax
+
+RUN mkdir -p /home/linuxbrew/repo-bare /home/linuxbrew/repo-local
+
+RUN cd /home/linuxbrew/repo-bare && git init --bare
+
+RUN cd /home/linuxbrew && git clone /home/linuxbrew/repo-bare /home/linuxbrew/repo-local
+
+RUN cd /home/linuxbrew/repo-local && \
+    git config --global user.email "demo@example.com" && \
+    git config --global user.name "Demo User" && \
+    git config --global init.defaultBranch master
```

Let's use main as default branch name.
```suggestion
    git config --global init.defaultBranch main
```